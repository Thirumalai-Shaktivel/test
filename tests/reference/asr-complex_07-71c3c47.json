{
    "basename": "asr-complex_07-71c3c47",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/complex_07.f90",
    "infile_hash": "98ceed61d8a2eaf04519347bac51e261cf6167038a3238a7016f4f0e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-complex_07-71c3c47.stdout",
    "stdout_hash": "1146294a206f0cb2b2569982a21f73ba16ada7487ab1ab7ac852a6b4",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}