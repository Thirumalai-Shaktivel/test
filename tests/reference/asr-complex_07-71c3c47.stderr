semantic error: Value of a save or parameter variable must evaluate to a compile time constant
 --> tests/../integration_tests/complex_07.f90:5:5
  |
5 |     complex :: x = cmplx(real_part)
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
